### DISCUSSION TITLE
Add CloudReady support, please.

### DISCUSSION THREAD

**Discussion by AndreRicardoJS** [ventoy/Ventoy#1006] [Ideas]
https://www.neverware.com/freedownload#intro-text

**Comment by HawkFest** [comment]
Please no! For a cloud-based remote software system owned by Google, who decides which specific brand of laptops is allowed - which is not much look-up their list (in a cartel-like scheme), more over under NSA's scrutiny (USA's Cloud-Act)? Ventoy's developers should get paid a mountain of money to become accomplices and lose their time on this, if not only for ethical matters... 😉

**Reply by chromer030** [reply]
CloudReady which is based on ChromiumOS has a multiple and various formatted internal filesystems with modified attributes and included grub bootloader in it's img file, it's not easy for a bootloader to mount an img that includes several filesystems and its own bootloader.

**Comment by ventoy** [comment]
@anrisilva 
CloudReady is supported since Ventoy-1.0.51
